Create introduction.md

xxxalxxx opened the pull request.

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

xxxalxxx committed:
Create introduction.md

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1 @@
+123123131321312
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

xxxalxxx committed:
Update introduction.md

xxxalxxx committed:
Update introduction.md

xxxalxxx committed:
Merge pull request #1 from xxxalxxx/deadly_skills

Update introduction.md

a-melnichuk committed:
initial commit

a-melnichuk committed:
resolve merge

xxxalxxx committed:
Update introduction.md

codeschool-kiddo committed:
Added new rule about coffee

xxxalxxx committed:
Merge pull request #3 from xxxalxxx/new_rules

Too many people are spilling coffee

a-melnichuk committed:
rules added

xxxalxxx committed:
Merge pull request #2 from xxxalxxx/deadly_skills

Deadly skills 101

a-melnichuk committed:
added useless file

a-melnichuk committed:
merging with tagged branch

ase_branch_1.0

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

xxxalxxx committed:
Update kill_list.md

a-melnichuk committed:
changed assassin names

a-melnichuk committed:
removed useless files

xxxalxxx committed:
Merge pull request #5 from xxxalxxx/release_branch_1.0

Release branch 1.0

a-melnichuk committed:
fixes #6

xxxalxxx committed:
fixes #6 too

xxxalxxx committed:
Merge pull request #7 from xxxalxxx/kill_list

Kill list

xxxalxxx committed:
Update README.md
